### ISSUE TITLE
archive.* links appear to be inaccessible

### ISSUE THREAD

**Issue by andrey-seryukov** [MaximilianKohler/HumanMicrobiome#1]
2019/07/31 - both archive.is and archive.fo links are down. It may be useful to have a "backup" list of links to original pages even if their content is transient. Original URLs are useful for lookups in alternative archive services, etc. PS: Thank you for making this wiki! :)

**Comment by MaximilianKohler** [comment]
If they're not working it's probably because you customized your DNS. Probably Cloudflare’s 1.1.1.1? Solution would be to change it back to default or contact Cloudflare to see if they can fix it on their end. 

That's a good point about having the original URLs available. I'm not entirely sure the best way to do that. Using archive.org instead would work since it keeps the original URL and just adds its own prefix. But I like archive.fo since it gives a short URL and you can use it to link to specific parts of long pages.

**Comment by andrey-seryukov** [comment]
Yes, Cloudflare it was. Changed it to ISP's default and all is working well.

**Comment by MaximilianKohler** [comment]
I'm going to leave this open so people see a "1" instead of "0" on the "issues" tab. Hopefully that will lead to more people clicking the issues tab to solve this when they run into it.
